Doc/library/sys.rst

```diff
@@ -1122,6 +1122,12 @@ always available.
    ``cache_tag`` is set to ``None``, it indicates that module caching should
    be disabled.
 
+   *arch* describes the architecture of the current interpreter runtime system.
```

Comment:
"current interpreter runtime system" is wrong--`_architecture` defines what architecture CPython was built with, not what it's currently running under.

Reply:
It's close. It specifies the architecture it was built _to_ run as, which is independent from both the one it was built _on_, and the one it's currently _running_ on.

But it's the only one that is specific to the currently running interpreter. So at most, I'd only change this to "the native architecture of the current interpreter", and possibly add "which will usually match the architecture of the current system" (though that's probably more words than it's worth).